### PR TITLE
Include bounding boxes, keypoints and skeletons in COCO format

### PR DESCRIPTION
**Pull request by andreaazzini**

### CODE REVIEW

File: darwin/dataset/upload_manager.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def add_files_to_dataset(
         }
         if path:
             payload["path"] = path
-        data = client.put(endpoint=f"/datasets/{dataset_id}/data", payload=payload, team=team,)
+        data = client.put(endpoint=f"/datasets/{dataset_id}/data", payload=payload, team=team)
```
Comment (andreaazzini) [inline darwin/dataset/upload_manager.py:62]: Formatter

File: darwin/torch/dataset.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ def __getitem__(self, index: int):
             # Extract the sequences of coordinates from the polygon annotation
             annotation_type = "polygon" if "polygon" in annotation else "complex_polygon"
             sequences = convert_polygons_to_sequences(
-                annotation[annotation_type]["path"], height=target["height"], width=target["width"],
+                annotation[annotation_type]["path"], height=target["height"], width=target["width"]
```
Comment (andreaazzini) [inline darwin/torch/dataset.py:174]: Formatter

File: darwin/torch/dataset.py [hunk 3]
```diff
@@ -251,7 +251,7 @@ def __getitem__(self, index: int):
         annotations = []
         for obj in target["annotations"]:
             sequences = convert_polygons_to_sequences(
-                obj["polygon"]["path"], height=target["height"], width=target["width"],
+                obj["polygon"]["path"], height=target["height"], width=target["width"]
```
Comment (andreaazzini) [inline darwin/torch/dataset.py:254]: Formatter

File: setup.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="darwin-py",
-    version="0.5.10",
+    version="0.5.11",
```
Comment (andreaazzini) [inline setup.py:8]: Is this the only thing I need to do to make it a new version?

**Comment by simedw** [comment]
close until mainfests are available

**Pull request closed without merging**